src/height/types.ts

```diff
@@ -0,0 +1,47 @@
+export interface IHeightConfig {
+  apiKey: string;
+}
+
+export interface IHeightQueryParams {
+  page?: number;
+  limit: number;
+  usePagination?: boolean;
```

Comment:
I think you don't need this anymore

Reply:
I think we better keep it. 
<img width="791" alt="image" src="https://github.com/user-attachments/assets/34befc5c-2396-45d4-9fc3-fa0e97ff2ff5">